styles.css

```diff
@@ -0,0 +1,595 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Helvetica, Arial, sans-serif;
+}
+
+/* -------------------- THE NAVBAR --------------- */
+
+header {
+  position: sticky;
+  top: 0;
+  background-color: #fff;
+  z-index: 30;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 15px;
+  border-bottom: 1px solid #e2e2e2;
+}
+
+.nyt-logo {
+  width: 195px;
+}
+
+.left {
+  color: black;
+}
+
+.left * {
+  margin: 0 5px;
+}
+
+.left-div a {
+  text-decoration: none;
```

Comment:
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
For example: Group all the rules that are common to all the anchor tags:
![image](https://user-images.githubusercontent.com/56762529/90825506-65ca6900-e30f-11ea-9619-83a671c99be7.png)